content/docs/sdk/flutter/key_basics.md

```diff
@@ -264,109 +370,4 @@ When an original key is updated by the creator, the ones who have cached it need
-
-Returns a boolean value that represents the status of the putMeta method. Returns ‘TRUE’ when putMeta is completed successfully, else returns false.
-
-Throws **AtClientException** with an error code and error description that describes that cause of the exception.
-
-**Read Data**
-
-**_get_**
-
-Get the value of the key from the user's cloud secondary.
-Signature
-`Future<AtValue> get (AtKey key);`
-
-Accepts an Instance of AtKey. The Metadata is encapsulated in the AtKey. Fetch both the value and metadata of the key.
-
-Returns a boolean value that represents the status of the ‘get’ method. Returns ‘TRUE’ when ‘get’ is completed successfully, else returns false.
-
-Throws **AtClientException** with an error code and error description that describes that cause of the exception.
-
-**_getMeta_**
-
-Gets the metadata of the key.
-
-**Signature**
-
-`Future <bool> getMeta(AtKey key);`
-
-Accepts an Instance of AtKey. The Metadata is encapsulated in the AtKey. Gets the metadata of the key.
-
-Returns a boolean value that represents the status of the putMeta method. Returns ‘TRUE’ when getMeta is completed successfully, else returns false.
-
-Throws _AtClientException_ with an error code and error description that describes that cause of the exception.
-
-**Delete Data**
-
-Deletes the key.
-
-**Signature**
-`Future<bool> delete(AtKey key);`
-
-Accepts an Instance of AtKey. Deletes the key from the storage. If the key is shared with another atSign, notifies the other atSign on the deletion of the key.
-
-Returns a boolean value that represents the status of the delete method. Returns ‘TRUE’ when delete is completed successfully, else returns false.
-
-Throws **AtClientException** with an error code and error description that describes that cause of the exception.
-
-**List of Keys**
-
-**_getKeys_**
-
-Get all the keys stored in the user's secondary in string format.
-
-_Signature_
-
-`Future <<List<String>> getKeys({String? regex, String? sharedBy, String? sharedWith});`
-
-If regex is specified only matching keys are returned, giving you the flexibility to use different regexes for different types of data.
-
-**_getAtKeys_**
-
-Get all the keys stored in the user's secondary in [AtKey] format.
-
-**Signature**
-
-`Future <<List<AtKey>> getAtKeys({String? regex, String? sharedBy, String? sharedWith});`
-
-If regex is specified only matching keys are returned, giving you the flexibility to use different regexes for different types of data.
-
-**Notification**
+CCD (Cascade delete) compliments the TTR functionality by allowing the owner of the key to decide if the cached key has to be deleted upon the deletion of the original key. CCD accepts a boolean value(either true (or) false). When set to true, upon deletion of the original key, the cached key is also deleted. Setting false results in the cached key remains intact even after the original key is deleted. 
```

Comment:
`Setting false...` -> `Setting it to false results in the cached key remaining, even after the original key is deleted.`